src/fieri/.env

```diff
@@ -0,0 +1,4 @@
+FIERI_FOODCRITIC_FAIL_TAGS=any
+FIERI_FOODCRITIC_TAGS=~FC031 ~FC045
+FIERI_RESULTS_ENDPOINT=http://supermarket.getchef.com/api/v1/cookbook-verisons/evaluation
+FIERI_KEY=YOUR_FIERI_KEY
```

Comment:
Results endpoint needs to be changed to local here, doesn't need to go out to public endpoint, it can just talk to itself. If we ever get to a point where we split up where the web app and workers run, the port will need to change.